flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "disable-external-data-checker": true
```

Comment:
why not removing the f-e-d-c stuff from the manifest instead?

Reply:
Could you elaborate on what you meant by f-e-d-c stuff? The reason to disable the check was that rust and go dependencies cannot be automatically upgraded since the versions of all the dependencies of dependencies need to be updated as well.

Reply:
why did you put the `x-checker-data`?

Reply:
I saw some other manifests that use it to update their dependencies, so I added them in this manifest. Although the auto-PR feature from flathub wouldn't be too useful, being able to check for updates for all dependencies by simply running `flatpak-external-data-checker` is still nice to have, in my opinion.